crates/lsp/src/text_edit.rs

```diff
@@ -79,12 +79,6 @@ impl TextEdit {
         builder.finish()
     }
 
-    // --- Start Posit
-    pub fn diff(text: &str, replace_with: &str) -> TextEdit {
-        super::diff::diff(text, replace_with)
-    }
-    // --- End Posit
```

Comment:
This is less important, but we can keep `text_edit.rs` a "pure" vendored file if we just reference `crate::diff::diff()` rather than trying for `TextEdit::diff()`. It doesn't matter too much either way i guess, this just felt better?